lib/suspenders/adapters/heroku.rb

```diff
@@ -34,6 +33,43 @@ def set_heroku_rails_secrets
         end
       end
 
+      def provide_review_apps_setup_script
+        app_builder.template(
+          "bin_setup_review_app.erb",
+          "bin/setup_review_app",
+          force: true,
+        )
+        app_builder.run "chmod a+x bin/setup_review_app"
+      end
+
+      def create_heroku_pipelines_config_file
+        template "app.json.erb", "app.json"
+      end
+
+      def create_heroku_pipeline
+        heroku_app_name = app_name.dasherize
+
+        pipelines_plugin = `heroku plugins | grep pipelines`
+        if pipelines_plugin.empty?
+          puts 'You need heroku pipelines plugin. Run: heroku plugins:install heroku-pipelines'
```

Comment:
Line is too long. [95/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.